src/app/meds-n-labels/activity-3/activity-3.component.html

```diff
@@ -1,9 +1,16 @@
-<div class="col-md-4 col-xs-6 col-md-offset-4 col-xs-offset-3">
+<div class="col-md-5 col-xs-6 col-md-offset-3 col-xs-offset-3" id="chatBox">
   <div class="chat-window">
     <H3 class="text-center stage-header">{{language?.heading}}</H3>
     <div class="chats" #chats>
       <div [className]="chat.status" *ngFor="let chat of messages">
         {{chat.message}}
+        <img [src]="chat.image" *ngIf="chat.image">
+        <div *ngIf="chat.link">
+            {{language?.activityIndicationMessage}}
+          <div *ngFor="let link of chat.link">
+            <a [href] = "link.href"><button mat-stroked-button class="link-button">{{link.name}}</button></a>
```

Comment:
using router link is not possible here ?

Reply:
@lunayach  please address this so that it can be merged.

Reply:
@shreyans29 We can only bind strings to routerLink. Here it was a property, **link.href**
That is, I checked for something like **[routerLink]**, It was not there.

Reply:
got it..